src/api/ingest/utils.py

```diff
@@ -11,11 +8,9 @@
     FishBeltCSVSerializer,
 )
 from api.models import (
-    BENTHICLIT_PROTOCOL,
```

Comment:
We need to leave these two imports in. They're not used currently, but they're about to be.

Reply:
Sounds good, committed.